### ISSUE TITLE
[Android][Accessibility] WCAG 2.5.5: Touch inputs must have a sufficient target size. (android.widget.RadioButton)

### ISSUE THREAD

**Issue by golddove** [microsoft/AdaptiveCards#4104]
## Platform
* Android

## Version
master

#### Issue
`Touch inputs must have a sufficient target size.` ([TouchSizeWcag](https://accessibilityinsights.io/info-examples/android/touch-size-wcag/))

![screenshot](https://user-images.githubusercontent.com/4442788/83663657-f73f0000-a596-11ea-9851-d9d10f3538f4.png)

#### Target application
io.adaptivecards.adaptivecardssample
#### Element path
android.widget.RadioButton
#### How to fix
    The element has an insufficient target size (width: 60.952dp, height: 19.428dp). Set the element's minWidth and minHeight attributes to at least 44dp.
#### Environment

This accessibility issue was found using Accessibility Insights for Android 2020.527.10 (axe-android), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
